### ISSUE TITLE
`schema` is marked as an invalid Sequelize constructor Options object param

### ISSUE THREAD

**Issue by orlando** [sequelize/sequelize#12606]
### What are you doing?
`schema` is missing from Sequelize constructor `Options` interface.

![image](https://user-images.githubusercontent.com/849872/89847252-1d5dbf00-db49-11ea-8068-ff63af69e6eb.png)

This can be replicated by creating a constructor and passing `schema` to options.
```
const connection = new Sequelize(
  databaseConfigEnv.database,
  databaseConfigEnv.username,
  databaseConfigEnv.password,
  {
    schema: databaseConfigEnv.schema,
  },
);
```

`schema` is missing in the `Options` interface definition https://github.com/sequelize/sequelize/blob/v6/types/lib/sequelize.d.ts#L179-L392, but it actually exists in the constructor options object https://github.com/sequelize/sequelize/blob/v6/lib/sequelize.js#L141.

### What do you expect to happen?

Passing `schema` to constructor options object shouldn't give a TypeScript error.

### What is actually happening?

Passing `schema` to constructor options object gives a TypeScript error.

```ts
Argument of type '{ schema: any; }' is not assignable to parameter of type 'Options'.
  Object literal may only specify known properties, and 'schema' does not exist in type 'Options'.
```

```
Output here
```

### Environment

- Sequelize version: 6.3.4 
- Node.js version: 12.8.2 
- Operating System: Catalina
- If TypeScript related: TypeScript version: 3.9.6

### How does this problem relate to dialects?

- [x] I think this problem happens regardless of the dialect.
- [ ] I think this problem happens only for the following dialect(s): 
- [ ] I don't know, I was using PUT-YOUR-DIALECT-HERE, with connector library version XXX and database version XXX

### Would you be willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time and I know how to start.
- [x] Yes, I have the time but I don't know how to start, I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

**Comment by gcherem** [comment]
Any update on it? I am having the same problem

**Comment by malkiz-rapid** [comment]
any updates? I also have this problem

**Comment by Grutula** [comment]
any updates? I also have this problem

**Comment by WikiRik** [comment]
This issue will be fixed with #13408
